_pages/packaging-resources.md

```diff
@@ -0,0 +1,145 @@
+---
+layout: splash
+classes: flowing
+permalink: /python-packaging-help.html
+title: "Python packaging help and resources"
+excerpt: "We make the scientific Python package ecosystem easier to navigate through peer review and packaging resources."
+header:
+  overlay_image: images/header.jpg
+  overlay_filter: 0.8
+intro:
+  - excerpt: "More here on community packaging."
+scientists:
+  - image_path: images/pyopensci-people-at-computer.jpg
+    alt: "A pencil sketch of a round table with people sitting around it from different backgrounds working on laptops and also writing together."
+    excerpt: "Our catalog of vetted open source tools makes it easier for scientists to find the trusted tools that they need to develop their open science workflows."
+    title: "Help scientists find the open science tools that they need"
+    url: https://www.pyopensci.org/python-packages.html
+    btn_label: View our growing list of accepted scientific Python packages
+    btn_class: btn--primary
+packaging-easier:
+   - image_path: images/people-building-blocks.jpg
+     alt: "A group of people working on a tetris like set of building blocks trying to stack them all together. "
+     title: "Contribute to our packaging resources"
+     excerpt: "More here about our review process with links :
+
+     * More
+
+     * More
+     "
+     url: https://www.pyopensci.org/python-package-guide
+     btn_label: View our Python packaging guide
+     btn_class: btn--primary
+partnership_benefits:
+  - image_path: /images/peer-review/pyos-partnerships-peer-review.png
+    alt: "Diagram with a light purple background. On the left there is the pyOpenSci purple flower and it says accepted with a check mark above. There are two arrows leading to boxes on the right. The top box says JOSS published with a check next to it and the JOSS logo. The box below says Community Affiliated with a check. THe boxes are numbered 1,2,3. "
+    excerpt: "
+Through a single review process, community maintainers:
+
+* Have their package accepted into the pyOpenSci ecosystem
+
+* Can be [published in JOSS (if in scope)](https://www.pyopensci.org/software-peer-review/partners/joss.html#)
+
+* Can become an affiliated package with your community, following your community guidelines.
+
+* Are supported in maintaining their tools, with input from our diverse, knowledgeable community, including active members from across the Python, Conda, PyPA and broader Python packaging ecosystem."
+    title: "Benefits of partnering with us"
+toc: false
+---
+
+{% include feature_row id="intro" type="center" %}
+
+<div class="pyos-section purple">
+<div class="content" markdown="1">
+
+## Check out our Python Packaging guidebook pages
+
+They've been reviewed.. etc etc...
```

Comment:
should there be more here?

Reply:
I'm so sorry, @kierisi ! I must have asked you to review this but it's still a draft PR. so there is absolutely missing text as you've noticed!! 🙃

Reply:
hahaha @lwasser no worries! I figured as much, but wanted to flag it just in case 😂